README.md

```diff
@@ -144,8 +117,8 @@ You can deploy this project using:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] As suggested by the previous reviewer, please kindly remove the descriptions underneath the headings telling you to `Describe 1 - 3 features you will add to the project.`